client/src/scenes/MainPage/index.jsx

```diff
@@ -13,262 +14,265 @@ const MainPage = () => {
   function handleSubmit() {
     setEmail('');
   }
+  const token = localStorage.getItem('token');
```

Comment:
I think it should be done with the isAuthorized flag in the redux state

Reply:
Agree